web/tls_config.go

```diff
@@ -40,9 +41,10 @@ var (
 )
 
 type Config struct {
-	TLSConfig  TLSConfig                     `yaml:"tls_server_config"`
-	HTTPConfig HTTPConfig                    `yaml:"http_server_config"`
-	Users      map[string]config_util.Secret `yaml:"basic_auth_users"`
+	TLSConfig         TLSConfig                     `yaml:"tls_server_config"`
+	HTTPConfig        HTTPConfig                    `yaml:"http_server_config"`
+	RateLimiterConfig RateLimiterConfig             `yaml:"rate_limiter_config"`
```

Comment:
I know the other options are `_config`, but I think it's a bit repetitive. How about:
```suggestion
	RateLimiterConfig RateLimiterConfig             `yaml:"rate_limit"`
```